packages/neuron-wallet/src/database/chain/ormconfig.ts

```diff
@@ -57,6 +59,7 @@ const connectOptions = async (genesisBlockHash: string): Promise<SqliteConnectio
     entities: [
       HdPublicKeyInfo,
       Transaction,
+      TxDescription,
```

Comment:
I'd like to know why not use glob patterns here?

Reply:
Do you mean the naming pattern?

Reply:
> Do you mean the naming pattern?

Yep

Reply:
Right. So perhaps TransactionDescription would be more concise.

Reply:
Actually, glob patterns or naming patterns I mentioned here are things like https://github.com/typeorm/typeorm/blob/master/README.md#L304

Reply:
```JSON
{
  "entities": [
    "src/entity/**/*.ts"
  ],
}
```

Reply:
Looks convenient to register entities in this way. We can do this in another PR dedicated for this.